File: basic_math/prime.py
def prime(n):
    c = 0
    for i in range(1, n + 1):
        if n % i == 0:
            c = c + 1
        
    if c == 2:
        return True
    else:
        return False


print(prime(2))
print(prime(25))
print(prime(1))

File: basic_math/Prime.java
package basic_math;

public class Prime {
    public static void main(String[] args) {
        System.out.println(prime(5));
        System.out.println(prime(25));
        System.out.println(prime(1));
    }
    public static boolean prime(int n){
        int c = 0;
        for ( int i =1; i<=n; i ++){
            if (n%i == 0){
                c = c + 1;
            }
        }
        if (c == 2){
            return true;
        }
        else {
            return false;
        }
    }

}
